### ISSUE TITLE
Single data point on victorybar chart

### ISSUE THREAD

**Issue by arpitjain-in** [FormidableLabs/victory-native#410]
Hi,
Every hour, I'm dynamically loading data to victory chart containing `VictoryAxis` and `VictoryBar`. At midnight 01 AM, it's the moment when I've only single data point in victory bar.

My issue is -
Bar is left aligned and requirement is to make it `center` align. I tried with all the ways but didn't succeed. Below is the snippet 

```
<VictoryChart 
    domainPadding={20} 
    scale={{ x: "time" }}>
<VictoryAxis/>
<VictoryAxis dependentAxis/>
<VictoryBar
  data={[
    { x: new Date("2018-11-09T00:00:00"), y: 2}
  ]}
/>
</VictoryChart>
```

Please help as I'm having delivery on my head.

Thanks

**Comment by arpitjain-in** [comment]
@boygirl Please help me as always. I'm having this as major defect.

**Comment by boygirl** [comment]
@arpitjain-in you're right. this is a bug. I will look into it. In the meantime, I think unfortunately you will have to add a domain manually when there is only one data point. I made a small example that should help you:

https://codesandbox.io/s/jj601wnz83

**Comment by boygirl** [comment]
@arpitjain-in 
I've fixed this bug in `victory@30.6.1`, and `victory-native@^30.0.0` should be able to pick up these changes with a fresh npm install.

**Comment by arpitjain-in** [comment]
Thanks a ton @boygirl. You are my life-savior.
